python/nano/tutorial/notebook/training/tensorflow/tensorflow_training_embedding_sparseadam.ipynb

```diff
@@ -0,0 +1,344 @@
+    "        with_info=True\n",
+    "    )\n",
+    "\n",
+    "    def custom_standardization(input_data):\n",
+    "        lowercase = tf.strings.lower(input_data)\n",
+    "        stripped_html = tf.strings.regex_replace(lowercase, \"<br />\", \" \")\n",
+    "        return tf.strings.regex_replace(\n",
+    "            stripped_html, f\"[{re.escape(string.punctuation)}]\", \"\"\n",
+    "        )\n",
+    "\n",
+    "    vectorize_layer = TextVectorization(\n",
+    "        standardize=custom_standardization,\n",
+    "        max_tokens=20000,\n",
+    "        output_mode=\"int\",\n",
+    "        output_sequence_length=500,\n",
+    "    )\n",
+    "    \n",
+    "    text_ds = raw_train_ds.map(lambda x, y: x)\n",
+    "    vectorize_layer.adapt(text_ds)\n",
+    "\n",
+    "    def vectorize_text(text, label):\n",
+    "        text = tf.expand_dims(text, -1)\n",
+    "        return vectorize_layer(text), label\n",
+    "\n",
+    "    # vectorize the data\n",
+    "    train_ds = raw_train_ds.map(vectorize_text)\n",
+    "    val_ds = raw_val_ds.map(vectorize_text)\n",
+    "    test_ds = raw_test_ds.map(vectorize_text)\n",
+    "\n",
+    "    return train_ds, val_ds, test_ds"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "train_ds, val_ds, test_ds = create_datasets()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; _The definition of_ `create_datasets` _can be found in the_ [runnable example](https://github.com/intel-analytics/BigDL/tree/main/python/nano/tutorial/notebook/training/tensorflow/tensorflow_training_embedding_sparseadam.ipynb)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We could then define the model. Same as using `tf.keras.layers.Embedding`, you could **instantiate a Nano's** `Embedding` **layer** as the first layer in the model:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "inputs = tf.keras.Input(shape=(None,), dtype=\"int64\")\n",
+    "\n",
+    "# 20000 is the vocabulary size,\n",
+    "# 128 is the embedding dimension\n",
+    "x = Embedding(minput_dim=20000, output_dim=128)(inputs)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "> 📝 **Note**\n",
+    ">\n",
+    "> If you would like to apply a regularizer function to the embedding matrix through setting `embeddings_regularizer`, Nano will apply the regularizer to the output tensors of the embedding layer instead to avoid making the sparse gradient dense (if `activity_regularize=None`).\n",
+    ">\n",
+    "> Please refer to [here](https://github.com/intel-analytics/BigDL/blob/main/python/nano/src/bigdl/nano/tf/keras/layers/embeddings.py) for more information on `bigdl.nano.tf.keras.layers.Embedding`."
```

Comment:
I provide link to our source code here for user to get more info on `bigdl.nano.tf.keras.layers.Embedding`, and also the source code link for `bigdl.nano.tf.optimizers.SparseAdam` later. Because we do not have related api docs for now.

Should we add related api docs for these two apis?